app/styles/variables.css

```diff
@@ -19,6 +19,7 @@
 
   --lego-max-width: 1100px;
   --lego-default-padding: 2rem;
+  --bdb-interested: #B33939;
```

Comment:
I think I prefer to use existing colours rather than introducing new ones

Reply:
The question is then what is the best way:
If we do choose to use variables, the color will change with the lightning mode:
<img width="346" alt="image" src="https://github.com/webkom/lego-webapp/assets/102583541/53d44128-5fec-417a-82dc-428e8522da1a">
<img width="321" alt="image" src="https://github.com/webkom/lego-webapp/assets/102583541/f23003bb-0cff-49d5-8b02-3195e93c8b85">

While in light mode "ikke interessert" is darker than "Interessert", in dark mode the opposite can be observed. While this may not be a big issue, if the user of the site remember states based on color, it may be difficult if they change lightning. 
  We can either go through with or we can use a different color over all. The reason we chose to use red was that we thought it was smart that states that are similar should also have similar color. 
As we are new to this, we are of course open to suggestions, so what do you think we should do?

Reply:
@ivarnakken What's your opinion on adding color variables that don't change based on theme? It'd make the solution to issues such at these a lot easier.

Reply:
"Ikke interessert" should be the danger color. "Interessert" shouldn't really be red at all, but rather a green successful color.

The colors here in the bdb were picked randomly by a webkomer many many years ago. Feel free to change.